src/main/resources/messages.properties

```diff
@@ -26,7 +26,9 @@ user.not-registered=User has not been registered. Some fields are invalid
 invalid.token=You must be authenticated
 token.not.start.with.bearer=JWT token is not starts with Bearer prefix
 token.signature.not.valid=JWT signature does not match locally computed signature. JWT validity cannot be asserted and should not be trusted.
-token.expired=JWT expired at
+access.token.expired=JWT expired at
```

Comment:
Why don't you set parameter into the text as you did below?

Reply:
The real message: "JWT expired at 2021-08-30T22:32:27Z. Current time: 2021-08-30T22:32:28Z, a difference of 1081 milliseconds.  Allowed clock skew: 0 milliseconds."